velox/substrait/SubstraitParser.h

```diff
@@ -49,6 +49,9 @@ class SubstraitParser {
   std::shared_ptr<SubstraitType> parseType(
       const ::substrait::Type& substraitType);
 
+  // Parse substraitType type such as i32
+  std::string parseType(const std::string& substraitType);
```

Comment:
Maybe change to `/// Parse substraitType type such as i32.` to be aligned with others.